src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.android.cs

```diff
@@ -18,6 +18,11 @@ internal void Show(Page sender, SnackBarOptions arguments)
 		{
 			var view = Platform.GetRenderer(sender).View;
 			var snackBar = AndroidSnackBar.Make(view, arguments.MessageOptions.Message, (int)arguments.Duration.TotalMilliseconds);
+			var anchorView = arguments.AnchorView?.GetRenderer()?.View;
+			if (!(anchorView is null))
```

Comment:
We don't use pattern matching for null check
```suggestion
			if (anchorView != null)
```

Reply:
Out of curiosity is it a convention or for other reason ? ie: is there a real difference or just the syntax ?

Reply:
Well, the Xamarin has this pattern forever. I don't know why they don't change to use the pattern matching